contracts/rust/adapter/src/sol_types.rs

```diff
@@ -16,42 +16,46 @@
 
 use alloy::sol;
 
+use crate::bindings::stake_table_v2::{
+    EdOnBN254,
+    BN254::{self, BaseField},
+};
```

Comment:
This addition is weird. revisit later. 
(EdOnBN254 should be imported already somewhere iirc, so are other types, plus importing them from `stake_table_v2` also seems strange)